spec/features/course/homepage_spec.rb

```diff
@@ -6,10 +6,23 @@
 
   with_tenant(:instance) do
     let(:course) { create(:course, :opened) }
-    let(:achievement_feed_notification) do
-      achievement = create(:course_achievement)
+    let(:feed_notifications) do
```

Comment:
There is only one kind of notification ( `feed` ) here, the difference is that each one has a different `activity` and activities are created using `traits` already.